src/errors.js

```diff
@@ -0,0 +1,49 @@
+// errno
+const errnos = {
+  ERRNO_INVALID_API_CLIENT_NAME : 100,
+  ERRNO_UNAUTHORIZED            : 401,
+  ERRNO_FORBIDDEN               : 403,
+  ERRNO_INTERNAL_ERROR          : 500
+};
+exports.errnos = errnos;
+
+// Error messages.
+const errors = {
+  BAD_REQUEST     : 'Bad Request',
+  FORBIDDEN       : 'Forbidden',
+  INTERNAL_ERROR  : 'Internal Server Error',
+  UNAUTHORIZED    : 'Unauthorized'
+};
+exports.errors = errors;
```

Comment:
Let's not anticipate any error code or message. For example,`ERRNO_INVALID_API_CLIENT_NAME` or `ERRNO_UNAUTHORIZED` will probably not be used here. Just clear this objects for now and we will be adding the errors as we need them.